allman-editorconfig

```diff
@@ -0,0 +1,1330 @@
+[*]
```

Comment:
@seeseemelk I would recommend taking a play through with this to see if everything fits the required style